src/components/TeamCard.js

```diff
@@ -1,12 +1,15 @@
 import React from "react";
-import { Card, CardHeader, CardBody } from "@trussworks/react-uswds";
+import { Card, CardHeader, CardBody, CardMedia } from "@trussworks/react-uswds";
 
-const Team = ({ name, info }) => {
+const Team = ({ name, info, photo }) => {
   return (
     <Card
       gridLayout={{ desktop: { col: 4 }, tablet: { col: 6 } }}
       containerProps={{ className: "bg-base-lightest border-0" }}
     >
+      <CardMedia>
+        <img src={photo} alt="team member" />
```

Comment:
Minor, but I would suggest replacing the hard-coded "team member" text from your alt tag and replacing it with a translated version